fossevents/events/management/commands/__init__.py

```diff
@@ -0,0 +1,3 @@
+# -*- coding: utf-8 -*-
+__version__ = '0.1.0'
+__version_info__ = tuple([int(num) if num.isdigit() else num for num in __version__.replace('-', '.', 1).split('.')])
```

Comment:
Line 2 and 3 is not needed to be put in every __init__.py, you can/should remove them.